### PR TITLE
Convert Tiago to FLU

### PR DESCRIPTION
**Pull request by lukicdarkoo**
- [x] (511) robots/pal_robotics/tiago++/protos/Tiago++.proto
- [x] (512) robots/pal_robotics/tiago_extensions/protos/TiagoGripper.proto
- [x] (513) robots/pal_robotics/tiago_extensions/protos/TiagoBody.proto
- [x] (514) robots/pal_robotics/tiago_extensions/protos/TiagoRightArm.proto
- [x] (515) robots/pal_robotics/tiago_extensions/protos/TiagoFrontArm.proto
- [x] (516) robots/pal_robotics/tiago_extensions/protos/TiagoLeftArm.proto
- [x] (517) robots/pal_robotics/tiago_extensions/protos/TiagoRightHey5.proto
- [x] (518) robots/pal_robotics/tiago_extensions/protos/TiagoLeftHey5.proto
- [x] (519) robots/pal_robotics/tiago_iron/protos/TiagoIron.proto
- [x] (520) robots/pal_robotics/tiago_titanium/protos/TiagoTitanium.proto
- [x] (521) robots/pal_robotics/tiago_steel/protos/TiagoSteel.proto
- [x] (522) robots/pal_robotics/tiago_base/protos/TiagoBase.proto
- [x] (198) projects/robots/pal_robotics/tiago++/worlds/tiago++.wbt
- [x] (199) projects/robots/pal_robotics/tiago_iron/worlds/tiago_iron.wbt
- [x] (200) projects/robots/pal_robotics/tiago_titanium/worlds/tiago_titanium.wbt
- [x] (201) projects/robots/pal_robotics/tiago_steel/worlds/tiago_steel.wbt
- [x] (202) projects/robots/pal_robotics/tiago_base/worlds/tiago_base.wbt (depends on https://github.com/cyberbotics/webots/pull/3647)

### CODE REVIEW

**Comment by lukicdarkoo** [comment]
@ad-daniel It seems the most `world update` failures are coming from the `develop` branch. Since the `feature-enu-plan` will not be merged soon into `develop`, I believe it is better to fix everything there.

**Comment by ad-daniel** [comment]
Yes, it will require removing all the `rotation 0 0 1 0` now that it has become the default value, but didn't expect there to be this many. World update can be ignored.

File: projects/robots/pal_robotics/tiago++/worlds/tiago++.wbt [hunk 1]
```diff
@@ -4,11 +4,10 @@ WorldInfo {
     "This simulation shows a Tiago++ making hello with both arms in an industrial environment. It can also be controlled using the arrows on the keyboard."
   ]
   title "Simulation of PAL Robotics' TIAGo++ robot"
-  coordinateSystem "NUE"
 }
 Viewpoint {
-  orientation 0.28882343295926544 -0.9512999432180493 -0.10774712342777662 5.502273164124365
-  position -0.9802615488886062 1.7304528074815935 2.665124377061189
+  orientation -0.2524577502324584 0.7482692115667424 0.6134804572020622 1.0056342481413238
+  position -14.436996591665887 -9.961169434327116 18.254293081593474
```
Comment (joachimhgg) [inline projects/robots/pal_robotics/tiago++/worlds/tiago++.wbt:10]:
```suggestion
  orientation -0.04537836798484944 0.015059461718461582 0.9988563542030363 2.5014830680385063
  position 2.956307784608618 -4.303908343005968 1.6938599568355999
```
the viewpoint is not the same than the 2020b version

File: projects/robots/pal_robotics/tiago_base/protos/TiagoBase.proto [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 PROTO TiagoBase [
   field  SFVec3f     translation      0 0.095 0
```
Comment (joachimhgg) [inline projects/robots/pal_robotics/tiago_base/protos/TiagoBase.proto:9]:
```suggestion
  field  SFVec3f     translation      0 0 0.095
```

File: projects/robots/pal_robotics/tiago_base/worlds/tiago_base.wbt [hunk 3]
```diff
@@ -4,11 +4,10 @@ WorldInfo {
     "This simulation shows a Tiago Base moving in an industrial environment using a Braitenberg algorithm using the information received by its lidar."
   ]
   title "Simulation of PAL Robotics' TIAGo Base robot"
-  coordinateSystem "NUE"
 }
 Viewpoint {
-  orientation 0.05732916548390367 0.9962878606918261 0.06421732961610703 4.655654070084208
-  position -6.255538309281095 0.4366268100324396 -1.0512025768441615
+  orientation 0.2573160410490516 -0.6499114813658986 -0.7151248292484507 5.2719007299888885
+  position -13.513125538997697 -11.336868950006574 15.557518701031217
```
Comment (joachimhgg) [inline projects/robots/pal_robotics/tiago_base/worlds/tiago_base.wbt:10]:
```suggestion
  orientation 0.04207463719185939 -0.37422533569250593 -0.9263828166751608 6.041567672773838
  position -6.552398220985782 0.7357475895306282 0.6006993871667716
```
to have the same viewpoint than 2020b

File: projects/robots/pal_robotics/tiago_iron/protos/TiagoIron.proto [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 PROTO TiagoIron [
   field  SFVec3f     translation      0 0.095 0
```
Comment (joachimhgg) [inline projects/robots/pal_robotics/tiago_iron/protos/TiagoIron.proto:9]:
```suggestion
  field  SFVec3f     translation      0 0 0.095
```

File: projects/robots/pal_robotics/tiago_steel/protos/TiagoSteel.proto [hunk 5]
```diff
@@ -7,7 +7,7 @@
 
 PROTO TiagoSteel [
   field  SFVec3f     translation      0 0.095 0
```
Comment (joachimhgg) [inline projects/robots/pal_robotics/tiago_steel/protos/TiagoSteel.proto:9]:
```suggestion
  field  SFVec3f     translation      0 0 0.095
```

File: projects/robots/pal_robotics/tiago_titanium/protos/TiagoTitanium.proto [hunk 6]
```diff
@@ -7,7 +7,7 @@
 
 PROTO TiagoTitanium [
   field  SFVec3f     translation      0 0.095 0
```
Comment (joachimhgg) [inline projects/robots/pal_robotics/tiago_titanium/protos/TiagoTitanium.proto:9]:
```suggestion
  field  SFVec3f     translation      0 0 0.095
```

**Review by joachimhgg — COMMENTED** [review]
I am not sure about your modification on `convert_nue_to_enu.py`. For example for the ratslife.wbt a rotation around z was needed (`field['value'] = rotation(field['value'], [0, 0, 1, -0.5 * math.pi])`)

File: projects/robots/pal_robotics/tiago++/worlds/tiago++.wbt (hunk 1 above)
Comment (lukicdarkoo) [reply projects/robots/pal_robotics/tiago++/worlds/tiago++.wbt:10]: It's not the same, but I prefer the new one :)

Comment (lukicdarkoo) [reply projects/robots/pal_robotics/tiago++/worlds/tiago++.wbt:10]: Actually not :)

**Comment by lukicdarkoo** [comment]
> I am not sure about your modification on `convert_nue_to_enu.py`. For example for the ratslife.wbt a rotation around z was needed (`field['value'] = rotation(field['value'], [0, 0, 1, -0.5 * math.pi])`)

I revert the script changes. The changes I did were enough for Tiago worlds, but since you have more samples maybe it is better that you change the script in the rat's life PR.

**Review by joachimhgg — APPROVED** [review]

**Pull request merged**